config/locales/gobierto_dashboards/views/ca.yml

```diff
@@ -13,6 +13,7 @@ ca:
         dates: Dates
         description: Visualització i anàlisi dels contractes adjudicats i les licitacions
           del %{entity_name}
+        download_data: Descarregueu les dades en format reutilitzable
```

Comment:
Descàrregueu...

Reply:
Not correct,  no es esdrújula si no plan

Reply:
Sorry, es de ver *Descàrrega*